gnocchi/tests/functional/gabbits/search.yaml

```diff
@@ -40,6 +40,21 @@ tests:
        - "expected a list for dictionary value @ data["
        - "'and']"
 
+    - name: search like id
+      POST: /v1/search/resource/generic
+      data:
+        like:
+          id: fa%
+      status: 400
```

Comment:
Can you check the details of the error here? So we're sure it's the right one caught and it's explicit.